data/json/professions.json

```diff
@@ -5158,5 +5158,81 @@
                 "panties"
             ]
         }
+    },
+        {
+        "type": "profession",
+        "ident": "bio_sci",
+        "name": "Mad Bionics Scientist",
+        "description": "Before all this you studied bionics. You tried your experiments on animals and yourself.", 
+        "traits": [
+            "ANIMALDISCORD",
+            "PSYCHOPATH"
+        ],
+        "skills": [
+            {"level": 4, "name": "electronics"},
+            {"level": 3, "name": "mechanics"}
+        ],
+        "points": 4,
+        "CBMs": [
+            "bio_alarm",
+            "bio_geiger",
+            "bio_armor_head",
+            "bio_batteries"
+        ],                
+        "entries": [
+            { "item": "goggles_nv", "charges": 50}
+        ],
+        "items": {
+            "both":[
+               "glasses_reading",
+               "coat_lab",
+               "gloves_rubber",
+               "pants",
+               "socks",
+               "boots_rubber",
+               "id_science" 
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "bra",
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "druggie",
+        "name": "Meth Cook",
+        "description": "You made your living off of making and selling drugs.  At least now there are less cops, excluding the rotten ones & bots.",
```

Comment:
Suggestion: Just ""You made your living off of making and selling drugs.  At least now there are less cops, excluding the rotten ones."  has a nicer ring to it I think.

Reply:
It's how it was originally.

Reply:
*fewer